tests/fsharpqa/Source/ErrorMessages/UnitGenericAbstractType/E_UnitGenericAbstractType1.fs

```diff
@@ -0,0 +1,9 @@
+// #ErrorMessages #UnitGenericAbstractType 
+//<Expects status="error" span="(7,21)" id="FS0017">The member 'Apply : int -> unit' is specialized with 'unit' but 'unit' can't be used as return type of an abstract method parameterized on return type.\.</Expects>
```

Comment:
the regex at end `.\.` is right? is not just `\.`?
like that parse any character followed by dot, instead of just a dot

But i just tried with this https://regex101.com/r/oG5sD1/1 online regex, if you remove the `.` before `\.` works.